test/org.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+var test = require('tap').test
+
+var server = require('./lib/server.js')
+var common = require('./lib/common.js')
+var client = common.freshClient()
+
+var URI = 'http://localhost:1337'
+var PARAMS = {
+  auth: {
+    token: 'foo'
+  },
+  org: 'myorg',
+  user: 'myuser'
+}
+
+test('org: add user', function (t) {
+  server.expect('PUT', '/-/org/myorg/user', function (req, res) {
+    t.equal(req.method, 'PUT')
+    onJsonReq(req, function (json) {
+      t.same(json, {user: 'myuser'})
+      res.statusCode = 200
+      res.json({result: 'anything'})
+    })
+  })
+
+  client.org('add', URI, PARAMS, function (err, data) {
+    t.same(err, null)
+    t.same(data, {result: 'anything'})
+    t.done()
+  })
+})
+
+test('org: rm user', function (t) {
+  server.expect('DELETE', '/-/org/myorg/user', function (req, res) {
+    t.equal(req.method, 'DELETE')
+    onJsonReq(req, function (json) {
+      t.same(json, {user: 'myuser'})
+      res.statusCode = 200
+      res.json({result: 'anything'})
+    })
+  })
+
+  client.org('rm', URI, PARAMS, function (err, data) {
+    t.same(err, null)
+    t.same(data, {result: 'anything'})
+    t.done()
+  })
+})
+
+test('org: rm user (400)', function (t) {
+  server.expect('DELETE', '/-/org/myorg/user', function (req, res) {
+    t.equal(req.method, 'DELETE')
+    onJsonReq(req, function (json) {
+      t.same(json, {user: 'myuser'})
+      res.statusCode = 400
+      res.json({result: 'anything'})
+    })
+  })
+
+  client.org('rm', URI, PARAMS, function (err, data) {
+    t.ok(err)
+    t.same(data, {result: 'anything'})
+    t.done()
+  })
+})
+
+test('org: ls', function (t) {
+  server.expect('GET', '/-/org/myorg/user', function (req, res) {
+    t.equal(req.method, 'GET')
+    onJsonReq(req, function () {
+      res.statusCode = 200
+      res.json({result: 'anything'})
```

Comment:
Could you have the `ls` test return a data structure that looks like what the registry will return for this?